Corona-Warn-App/src/device/java/de/rki/coronawarnapp/ui/main/MainFragment.kt

```diff
@@ -159,6 +159,25 @@ class MainFragment : Fragment(R.layout.fragment_main) {
         popup.show()
     }
 
+    /**
+     * Checks if the interoperability feature was shown at least once to the user
+     * if not displays it otherwise nothing happens
+     */
+    private fun checkShouldInteroperabilityOpened() {
+        if (!LocalData.interoperabilityWasShown()) {
+            showInteroperabilityFeature()
+        }
+    }
+
+    /**
+     * Navigates to the delta onboarding screen for interoperability
+     */
+    private fun showInteroperabilityFeature() {
```

Comment:
How about renaming the function to navigateTo... and dropping the comment

Reply:
Fun fact: It was named navigateToInteroperability but I thought later it's better like explicit saying we want to show a feature. But I am open for this we could also change it back :)

Reply:
No strong feelings on this. Maybe we just stick to other naming patterns within the app.

Reply:
Renamed it. All other navigation or openings are named navigateTo